### DISCUSSION TITLE
Having issues with wipe archive feature, please help.

### DISCUSSION THREAD

**Discussion by novusB** [victornpb/undiscord#605] [1. Questions & Answers]
Good morning,

I am trying to wipe all of my data, I have requested and received my data from discord. I am attempting to use the 'wipe archive' feature. I am getting this response

`Fetching messages...
Error searching messages, API responded with status 403!
	{"message":"Missing Access","code":50001}
CoreException	{}`

What would be the resolution to this?

Thank you,

**Comment by novusB** [comment]
I was able to figure out the issue, I needed to edit my index.json file to remove 'None in' channels/servers, also to delete 'none' users who have deleted their accounts.

**Reply by ghost** [reply]
Hey Novus, I'm having issues with the wipe archive feature and I didn't really understand what you said. Could you plz help me to get it to work? thank you
